models/qualtrics__directory.sql

```diff
@@ -88,13 +88,13 @@ final as (
         agg_contacts.count_distinct_phones,
         agg_contacts.total_count_contacts,
         agg_contacts.total_count_unsubscribed_contacts,
-        agg_contacts.count_contacts_created_30d,
-        agg_contacts.count_contacts_unsubscribed_30d,
-        agg_distributions.count_contacts_sent_survey_30d,
-        agg_distributions.count_contacts_opened_survey_30d,
-        agg_distributions.count_contacts_started_survey_30d,
-        agg_distributions.count_contacts_completed_survey_30d,
-        agg_distributions.count_surveys_sent_30d,
+        agg_contacts.count_contacts_created_30d, 
+        agg_contacts.count_contacts_unsubscribed_30d, 
+        agg_distributions.count_contacts_sent_survey_30d, 
+        agg_distributions.count_contacts_opened_survey_30d, 
+        agg_distributions.count_contacts_started_survey_30d, 
+        agg_distributions.count_contacts_completed_survey_30d, 
+        agg_distributions.count_surveys_sent_30d, 
```

Comment:
A number of these are showing as null in our data. Can we coalesce to 0. Or do you feel it could cause confusion if we are saying `0` when there has actually been no activity 🤔. What are your thoughts?

Reply:
ah yeah meant to coalesce everything! except for avg's and medians